File: C#/Programming Basics with C#/11. For-Loop - Exercise/07. Salary/Salary.cs
using System;

class Salary
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        int salary = int.Parse(Console.ReadLine());
        for (int i = 1; i <= n; i++)
        {
            string saitName = Console.ReadLine();
            if (saitName == "Facebook")
            {
                salary = salary - 150;
            }
            else if (saitName == "Instagram")
            {
                salary = salary - 100;
            }
            else if (saitName == "Reddit")
            {
                salary = salary - 50;
            }
            if (salary <= 0)
            {
                break;
            }
        }

        if (salary <= 0)
        {
            Console.WriteLine("You have lost your salary.");
        }
        else
        {
            Console.WriteLine(salary);
        }
    }
}

File: C++/Programming Basics with C++/11. Exercise - For-loop/simple_tasks/Salary.cpp
#include <iostream>

using namespace std;

int main() {
    string input;
    getline(cin, input);
    int n = stoi(input);
    getline(cin, input);
    int salary = stoi(input);

    for (int i = 1; i <= n; i++) {
        string domain;
        getline(cin, domain);

        if (domain == "Facebook") {
            salary = salary - 150;
        } else if (domain == "Instagram") {
            salary = salary - 100;
        } else if (domain == "Reddit") {
            salary = salary - 50;
        }

        if (salary <= 0) {
            break;
        }
    }

    if (salary <= 0) {
        cout << "You have lost your salary." << endl;
    } else {
        cout << salary << endl;
    }

    return 0;
}
